packages/core/src/middleware/koa-connector-error-handle.ts

```diff
@@ -22,7 +22,7 @@ export default function koaConnectorErrorHandler<StateT, ContextT>(): Middleware
           throw new RequestError(
             {
               code: 'connector.invalid_config',
-              status: 500,
+              status: 400,
```

Comment:
Why 400? Invalid config should be an internal error, thus 500?
400 indicates a bad request

Reply:
`400` represents a user error.

Reply:
is the user's fault, user pass in an invalid config, throw 400 and reject to save.

Reply:
<img width="722" alt="image" src="https://user-images.githubusercontent.com/12833674/157604553-013e61a4-d5be-4b44-a7d0-60120a87d7ed.png">

After digging into this, I think 400 serves our purpose. So I'm OK with 400 now